### PR TITLE
[FE#101] Add filter on departments

### PR DESCRIPTION
**Pull request by boris-arkenaar**
closes Signalen/frontend#101

depends on Amsterdam/signals#706 (can be merged already though)

Adds a filter on departments.

https://user-images.githubusercontent.com/2932084/107955329-f079e100-6f9d-11eb-8cc5-bdbc0b301137.mov

### CODE REVIEW

**Review by aditudorache — APPROVED** [review]

File: src/signals/incident-management/containers/FilterTagList/FilterTagList.test.js [hunk 1]
```diff
@@ -282,6 +283,95 @@ describe('signals/incident-management/containers/FilterTagList', () => {
       });
     });
 
+    describe('routing departments', () => {
+      it('renders correctly', () => {
+        configuration.featureFlags.assignSignalToDepartment = true;
+        const { queryAllByTestId } = render(
```
Comment (MauritsR) [inline src/signals/incident-management/containers/FilterTagList/FilterTagList.test.js:289]: You can use `screen.queryAllByTestId` instead

File: src/signals/incident-management/components/FilterForm/__tests__/FilterForm.test.js [hunk 2]
```diff
@@ -248,6 +249,113 @@ describe('signals/incident-management/components/FilterForm', () => {
...
+      const asc = screen.getByLabelText(ascName);
+      const aeg = screen.getByLabelText(aegName);
+
+      userEvent.click(asc);
+      expect(asc).toBeChecked();
+      userEvent.click(aeg);
+      expect(aeg).toBeChecked();
+
+      userEvent.click(clearButton);
+      // expect(asc).not.toBeChecked();
```
Comment (MauritsR) [inline src/signals/incident-management/components/FilterForm/__tests__/FilterForm.test.js:345]: These can be uncommented, right?

Comment (boris-arkenaar) [reply src/signals/incident-management/components/FilterForm/__tests__/FilterForm.test.js:366]: They should, yes, but then the test fails, while it should pass.

File: src/utils/__tests__/fixtures/directingDepartments.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+[
```
Comment (aditudorache) [inline src/utils/__tests__/fixtures/directingDepartments.json:1]: This is not needed, we already have a mock for that in `src/utils/__tests__/fixtures/index.js`

**Review by aditudorache — CHANGES_REQUESTED** [review]

File: src/utils/__tests__/fixtures/directingDepartments.json (hunk 3 above)
Comment (boris-arkenaar) [reply src/utils/__tests__/fixtures/directingDepartments.json:1]: I prefer to have my fixtures explicitly defined and not calculated which adds unnecessary complexity, execution time, and possible points of failure.

Comment (aditudorache) [reply src/utils/__tests__/fixtures/directingDepartments.json:1]: Will keep this construction. The categories_private will be shorten.

**Review by aditudorache — APPROVED** [review]

**Review by MauritsR — APPROVED** [review]

**Pull request merged**